chart/templates/_helpers.yaml

```
@@ -733,20 +737,22 @@ fsGroup: {{ $.Values.gid }}
 {{- end -}}
 
 {{/*
-Set the default value for securityContext
-If no value is passed for securityContext or <node>.securityContext, defaults to UID in the local node.
+Set the default value for pod securityContext
+If no value is passed for <node>.securityContexts.pod or <node>.securityContext, defaults to UID in the local node.
 
-    +------------------------+     +-------------+
-    | <node>.securityContext |  >  | <node>.uid  |
-    +------------------------+     +-------------+
+    +------------------------+           +-----------------+            +-------------+
+    | <node>.securityContexts.pod |  ->  | <node>.securityContext |  >  | <node>.uid  |
+    +------------------------+           +-----------------+            +-------------+
```

```suggestion
    +-----------------------------+      +------------------------+      +-------------+
    | <node>.securityContexts.pod |  ->  | <node>.securityContext |  ->  | <node>.uid  |
    +-----------------------------+      +------------------------+      +-------------+
```

(I'm not going to do the rest via suggestions, but you get the idea)